plugin/web/extensions/sidebar/core/components/mobile/hooks/projectHooks.ts

```diff
@@ -283,9 +289,16 @@ export default ({
   const fetchedSharedProject = useRef(false);
 
   useEffect(() => {
-    if (!backendURL || !backendProjectName || fetchedSharedProject.current) return;
+    if (
+      (!isCustomProject && (!plateauBackendURL || !plateuProjectName)) ||
+      (isCustomProject && (!customBackendURL || !customBackendProjectName)) ||
+      fetchedSharedProject.current
+    )
+      return;
     if (projectID) {
       (async () => {
+        const backendURL = isCustomProject ? customBackendURL : plateauBackendURL;
```

Comment:
I don't think we need isCustomProject. Just prioritize the customBackendURL (`customBackendURL ?? plateauBackendURL`)